holoviews/operation/normalization.py

```diff
@@ -175,3 +177,29 @@ def _normalize_raster(self, raster, key):
                 if range:
                     norm_raster.data[:,:,depth] /= range
         return norm_raster
+
+
+class normalize_group(Operation):
```

Comment:
I think this should say something about subcoordinates as the output will only work in that case...horrible name but semantically it is something like `subcoordinate_group_ranges`